Content guidelines.md

```diff
@@ -0,0 +1,65 @@
+| --------------------------------------------------------------------------------------------- | --------------------------------------------------------------------------------------------------- |
+| ![Example of how to communicate external errors](images/do-communicating-external-errors.png) | ![Example of how not to communicate external errors](images/dont-communicating-external-errors.png) |
+
+**Best practices:**
+
+1. Make it clear that the issue requires contacting specialists on your platform.
+2. Provide an easy way for merchants to contact you.
+3. Be as clear as possible about what happened.
+
+---
+
+## Action required outside of Shopify
+
+While merchants are setting up their channel they may need to take an action outside of Shopify. Make sure to be as clear as possible outlining merchant’s next steps.
+
+|                                                                                                                         |                                                                                                                               |
+| ----------------------------------------------------------------------------------------------------------------------- | ----------------------------------------------------------------------------------------------------------------------------- |
+| ![Example of how to indicate that action is required ourside of Shopify](images/do-action-required-outside-shopify.png) | ![Example of how not to indicate that action is required outside of Shopify](images/dont-action-required-outside-shopify.png) |
+
+**Best practices:**
+
+1. Tell merchants where they need to go to resolve the issue, and be clear about when they are leaving Shopify.
+2. Provide a direct link or guide them.
+3. Tell merchants what they need to do after they leave Shopify.
+
+---
+
+## Important channel updates
+
+If there are new requirements for your channel or important information that merchants need to know, communicate them as early and as directly as possible. Mention the impact on the merchant’s store or sales, if any. Don’t rely on email or external communication, as merchants often miss it.
+
+|                                                                                           |                                                                                                 |
+| ----------------------------------------------------------------------------------------- | ----------------------------------------------------------------------------------------------- |
+| ![Example of how to communicate channel updates](images/do-important-channel-updates.png) | ![Example of how not to communicate channel updates](images/dont-important-channel-updates.png) |
+
+**Best practices:**
+
+1. Tell the merchants if any action is required of them.
+2. When possible, add a relevant date or time.
```

Comment:
```suggestion
2. Where possible, add a relevant date or time.
```

Reply:
Should we instantiate with an example? It's a little ambiguous as currently phrased, but it might be clear to devs

Reply:
updated the content here a bit for clarity